k8s/arroyo/templates/tests/test-grpc-connection.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "arroyo.fullname" . }}-test-connection"
+  labels:
+    {{- include "arroyo.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: grpcurl
+      image: fullstorydev/grpcurl:v1.8.7-alpine
+      args: ['-plaintext', '{{ include "arroyo.fullname" . }}-api:{{ .Values.service.grpcPort }}','list']
```

Comment:
I think we'll need to change this to the HTTP endpoint now as the gRPC endpoint is gone in the API server; we might also want to test that the other services are up

Reply:
There was already a test for the HTTP endpoint, should this just be removed?

Reply:
This should address the problem with the services  [`554cab7` (#271)](https://github.com/ArroyoSystems/arroyo/pull/271/commits/554cab74c3ae4a7eece9254e4d92da1a302323c7) 
I will submit a separate commit for the output dir